x-team/xp-ui
# Create work experience presenter component

**Release Type:** *Non-Breaking Feature*

Fixes https://x-team-internal.atlassian.net/browse/XP-3735

## Description

The WorkExperienceEntryPresenter component is fully implemented to render a single work experience entry for consumption on the admin dashboard. The basic framework for using InlineEditor to allow inline editing and creation of work experience entries is also in place, though the editor component is not coming until we finish XP-3736.

Stories are included for all possible prop combinations with the control.

## Checklist



## Steps to Test or Reproduce

Launch storybook with `npm run start` in xp-ui, then navigate to UI Controls > Work Experience > Presenter.

The three main scenarios are combinations of props which are required by spec. The various scenarios under "Debug" are there to handle missing props which hypothetically should never happen.

## Impacted Areas in Application

XP Admin UI

## Screenshots

![image](https://user-images.githubusercontent.com/5854132/80960194-12d8ae80-8e11-11ea-99ce-ee4e0bfa8112.png)
